Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -118,9 +119,16 @@ var TouchableNativeFeedback = React.createClass({
      *
      * @param color The ripple color
      * @param borderless If the ripple can render outside it's bounds
+     * @param borderWithRadius If the ripple should be contained with the target's border-radius, if defined. Only
+     * used if {@code borderless} is false.
      */
-    Ripple: function(color: string, borderless: boolean) {
-      return {type: 'RippleAndroid', color: processColor(color), borderless: borderless};
+    Ripple: function(color: string, borderless: boolean, borderWithRadius: boolean) {
+      return {
+        type: 'RippleAndroid',
+        color: processColor(color),
+        borderless: borderless,
+        borderWithRadius: borderWithRadius
```

Comment:
I can't think of a great use case. But I added it because the last commit was reverted because it wasn't opt-in.
<img width="880" alt="screen shot 2017-01-17 at 11 08 55 am" src="https://cloud.githubusercontent.com/assets/1754153/22123940/a17c5f56-de42-11e6-8d44-152aed7a18f7.png">

Though, I played around with adding borders and modals (which are the referenced reasons) and didn't see any weird behavior.